### DISCUSSION TITLE
[Java] Support RoadMap ?

### DISCUSSION THREAD

**Discussion by HichemMaiza** [microsoft/onnxruntime-genai#368] [New features / APIs]
Hello Guys, is there any plans to support Java along with the already supported bindings ? 

- Python
- C#
- C/C++

**Comment by natke** [comment]
Yes, there is a plan. We are working on the schedule for this. Converting over to a discussion

**Comment by natke** [comment]
We have added the Java API: https://github.com/microsoft/onnxruntime-genai/tree/main/src/java and packages in progress
